mknotes.sh

```
@@ -0,0 +1,8 @@
+#!/usr/bin/env zsh
```

Shellcheck (0.10.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)